x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_flyout/alerts_flyout.tsx

```diff
@@ -56,26 +56,91 @@ export const AlertsFlyout: React.FunctionComponent<AlertsFlyoutProps> = ({
   let Body: AlertTableFlyoutComponent;
   let Footer: AlertTableFlyoutComponent;
 
+  const {
+    header: internalHeader,
+    body: internalBody,
+    footer: internalFooter,
+  } = alertsTableConfiguration?.useInternalFlyout != null
+    ? alertsTableConfiguration?.useInternalFlyout()
+    : {
+        header: null,
+        body: null,
+        footer: null,
+      };
```

Comment:
nitpick: do we need to do all of this null check here? If so I think we can simplify this to

```
const { ... } = alertsTableConfiguration?.useInternalFlyout?.() || {}; (or default to something static like DEFAULT_INTERNAL_FLYOUT)
```

Reply:
yes we want to create default component but for now I think that's what we can do